app/views/departureP5/drafts/DeleteDraftDepartureYesNoView.scala.html

```diff
@@ -42,8 +43,9 @@
             field = form("value"),
             yesNoType = YesNoWithAdditionalHtml(
                 heading = messages("departure.drafts.deleteDraftDepartureYesNo.heading"),
-                additionalHtml = Html(s"""<p class="govuk-body">${messages("departure.drafts.deleteDraftDepartureYesNo.paragraph")}</p><dl class="govuk-summary-list"><div class="govuk-summary-list__row"><dt class="govuk-summary-list__key">Local reference number</dt><dd class="govuk-summary-list__value">$lrn</dd></div></dl>""")
-
+                additionalHtml =
+Html(s"""<p class="govuk-body">${messages("departure.drafts.deleteDraftDepartureYesNo.paragraph")}</p><div class="govuk-body">${govukInsetText(InsetText(content = Text(messages("departure.drafts.dashboard.table.action.delete.hidden", lrn))))}
+</div>""")
```

Comment:
To tidy it up a little you can mark this up separately like https://github.com/hmrc/ctc-departure-route-details-frontend/blob/59673e042372ebdb9a416434e99d32392e020ff2/app/views/locationOfGoods/AddLocationOfGoodsView.scala.html#L32-L35